frontend/public/components/RBAC/bindings.tsx

```
@@ -784,52 +788,79 @@ const getSubjectIndex = () => {
+
+  if (loadError) {
+    return <StatusBox data={obj} loaded={loaded} loadError={loadError} />;
+  }
+
+  if (!obj || _.isEmpty(obj)) {
+    return <StatusBox data={obj} loaded={loaded} loadError={loadError} />;
+  }
+
   const fixed: { [key: string]: any } = {};
-  _.each(props.fixedKeys, (k) => (fixed[k] = _.get(props.obj.data, k)));
+  fixedKeys.forEach((k) => (fixed[k] = obj?.[k]));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)